include/swift/AST/Attr.def

```diff
@@ -390,6 +390,9 @@ DECL_ATTR(_private, PrivateImport,
   OnImport |
   UserInaccessible |
   NotSerialized, 82)
+SIMPLE_DECL_ATTR(_alwaysEmitIntoClient, AlwaysEmitIntoClient,
+  OnVar | OnSubscript | OnAbstractFunction,
+  83)
```

Comment:
You'll need to bump SWIFTMODULE_VERSION_MINOR, right?

Reply:
You're right -- thanks!